README.md

```diff
@@ -24,6 +25,20 @@
 * Cereal (<https://github.com/klepsydra-technologies/cereal>)
 * Spdlog (<https://github.com/klepsydra-technologies/spdlog>)
 
+<!---
+FIXME Can we discard the (unrelated) source code and compile just with distro packages?
+      Remove from .gitmodules?
+
+```bash
+sudo apt-get install googletest googletest-tools google-mock
+sudo apt-get install libconcurrentqueue-dev
+sudo apt-get install libcereal-dev
+sudo apt-get install libspdlog-dev
```

Comment:
At the moment the distro packages do not correspond to the version we use on all the OS that we support. So we will not do this change at the moment.